utility/utils.py

```diff
@@ -750,7 +749,63 @@ def download_openshift_installer(version=defaults.INSTALLER_VERSION):
             f"https://mirror.openshift.com/pub/openshift-v4/clients/ocp/"
             f"{version}/openshift-install-{os_type}-{version}.tar.gz"
         )
+        # Prepare BIN_DIR
+        try:
+            os.mkdir(defaults.BIN_DIR)
+        except FileExistsError:
+            pass
+        # record current working directory and switch to BIN_DIR
+        previous_dir = os.getcwd()
+        os.chdir(defaults.BIN_DIR)
+        tarball = f"{installer_filename}.tar.gz"
+        download_file(url, tarball)
+        run_cmd(f"tar xzvf {tarball}")
+        os.remove(tarball)
+        # return to the previous working directory
+        os.chdir(previous_dir)
+
+    return installer_binary_path
+
+def get_openshift_client(version=defaults.CLIENT_VERSION):
+    """
+    Get path of the openshift client binary, download it if not available.
+
+    Args:
+        version (str): version of the client to download
+
+    Returns:
+        str: path of the client binary
+
+    """
+    client_binary_path = os.path.join(defaults.BIN_DIR, 'oc')
+    if os.path.isfile(client_binary_path):
+        log.info("Client exists, skipping download")
```

Comment:
I would only print in case the client is not exist

Reply:
I've changed to print this message only at *debug* level, is it ok?